dev-utils/build.js

```diff
@@ -168,7 +168,10 @@ function getWebpackEnv(env = 'development') {
   return {
     APM_SERVER_URL: serverUrl,
     STACK_VERSION: stackVersion,
-    NODE_ENV: env
+    NODE_ENV: env,
```

Comment:
Defaults to all variables passed through process.env since webpack 5 removed polyfilling `node` by default.